app/src/main/java/com/github/se/project/ui/profile/ProfileInfo.kt

```diff
@@ -47,42 +57,60 @@ fun ProfileInfoScreen(
   val profileState = listProfilesViewModel.currentProfile.collectAsState()
   val lessons = lessonViewModel.currentUserLessons.collectAsState()
 
-  // Filter only lessons that are marked as COMPLETED
+  // SectionInfo for Completed Lessons
+  val completedLessonsSection =
+      SectionInfo(
+          title = "Completed Lessons",
+          status = LessonStatus.COMPLETED,
+          icon = Icons.Default.CheckCircle)
+
+  // Filter completed lessons
   val completedLessons =
       lessons.value.filter {
         it.status == LessonStatus.COMPLETED || it.status == LessonStatus.PENDING_REVIEW
       }
-
   Scaffold(
       topBar = {
         Row(
-            modifier = Modifier.fillMaxWidth().padding(4.dp).testTag("profileTopBar"),
-            horizontalArrangement = Arrangement.SpaceBetween) {
-              Row(Modifier.padding(horizontal = 4.dp)) {
+            modifier =
+                Modifier.fillMaxWidth()
+                    .background(MaterialTheme.colorScheme.background)
+                    .padding(vertical = 16.dp, horizontal = 16.dp)
+                    .testTag("profileTopBar"),
+            horizontalArrangement = Arrangement.SpaceBetween,
+            verticalAlignment = Alignment.CenterVertically) {
+              Row(verticalAlignment = Alignment.CenterVertically) {
                 if (profileState.value?.role == Role.TUTOR) {
-                  IconButton(
-                      onClick = { navigationActions.navigateTo(Screen.EDIT_SCHEDULE) },
-                      modifier = Modifier.testTag("calendarButton")) {
-                        Icon(imageVector = Icons.Default.DateRange, contentDescription = "Calendar")
-                      }
```

Comment:
I think removing this button is a shame, while there is an access through the profile editing screen now, more paths is usually better, especially if they are shortcuts